cmd/runhcs/shim.go

```diff
@@ -180,14 +180,19 @@ var shimCommand = cli.Command{
 				Environment:      environment,
 				User:             spec.User.Username,
 			}
-			for i, arg := range spec.Args {
-				e := windows.EscapeArg(arg)
-				if i == 0 {
-					wpp.CommandLine = e
-				} else {
-					wpp.CommandLine += " " + e
+
+			if spec.CommandLine != "" {
+				wpp.CommandLine = spec.CommandLine
+				// Add optional subsequent arguments escaped. Caters for things like when
```

Comment:
This seems odd. Why can't Docker just do this step?

Reply:
@jstarks - that's resolved now. It's either CommandLine or Args. No mix/match